### PR TITLE
Update sql-statement-import-into.md

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-import-into.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 
 `IMPORT INTO` 语句使用 TiDB Lightning 的[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)，用于将 `CSV`、`SQL`、`PARQUET` 等格式的数据导入到 TiDB 的一张空表中。
 
-`IMPORT INTO` 支持导入存储在 Amazon S3、GCS、Azure Blob Storage 和 TiDB 本地的数据文件。
+`IMPORT INTO` 支持导入存储在 Amazon S3、GCS、和 TiDB 本地的数据文件。
 
-- 对于存储在 S3、GCS 或 Azure Blob Storage 的数据文件，`IMPORT INTO` 支持通过 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)运行。
+- 对于存储在 S3、GCS 的数据文件，`IMPORT INTO` 支持通过 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)运行。
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:12]:
```suggestion
- 对于存储在 S3、GCS 的数据文件，`IMPORT INTO` 支持通过 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)运行。
```
```suggestion
- 对于存储在 S3 或 GCS 的数据文件，`IMPORT INTO` 支持通过 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)运行。
```

File: sql-statements/sql-statement-import-into.md [hunk 2]
```diff
@@ -88,9 +89,9 @@ SET 表达式左侧只能引用 `ColumnNameOrUserVarList` 中没有的列名。
 
 ### fileLocation
 
-用于指定数据文件的存储位置，该位置可以是 S3、GCS 或 Azure Blob Storage URI 路径，也可以是 TiDB 本地文件路径。
+用于指定数据文件的存储位置，该位置可以是 S3、GCS 的 URI 路径，也可以是 TiDB 本地文件路径。
 
-- S3、GCS 或 Azure Blob Storage URI 路径：配置详见[外部存储服务的 URI 格式](/external-storage-uri.md)。
+- S3、GCS 的 URI 路径：配置详见[外部存储服务的 URI 格式](/external-storage-uri.md)。
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:94]:
```suggestion
- S3 或 GCS 的 URI 路径：配置详见[外部存储服务的 URI 格式](/external-storage-uri.md)。
```

File: sql-statements/sql-statement-import-into.md [hunk 3]
```diff
@@ -146,6 +147,7 @@ SET 表达式左侧只能引用 `ColumnNameOrUserVarList` 中没有的列名。
 > **注意：**
 >
 > Snappy 压缩文件必须遵循[官方 Snappy 格式](https://github.com/google/snappy)。不支持其他非官方压缩格式。
+> 无法对单个大压缩文件进行并发解压，因此压缩文件的大小会直接影响导入速度。建议压缩数据文件解压后的源文件大小不超过 256 MiB。
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:150]:
```suggestion
> - Snappy 压缩文件必须遵循[官方 Snappy 格式](https://github.com/google/snappy)。不支持其他非官方压缩格式。
> - 由于无法对单个大压缩文件进行并发解压，因此压缩文件的大小会直接影响导入速度。建议解压后的文件大小不要超过 256 MiB。
```

File: sql-statements/sql-statement-import-into.md [hunk 4]
```diff
@@ -245,7 +247,7 @@ IMPORT INTO t(id, name, @1) FROM '/path/to/file.csv' WITH skip_rows=1;
 IMPORT INTO t FROM '/path/to/file-*.csv'
 ```
 
-### 从 S3、GCS 或 Azure Blob Storage 导入数据
+### 从 S3、GCS 导入数据
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:250]:
```suggestion
### 从 S3 或 GCS 导入数据
```

File: sql-statements/sql-statement-import-into.md [hunk 5]
```diff
@@ -32,6 +32,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - 当使用[全局排序](/tidb-global-sort.md)导入数据时，单行数据的总长度不能超过 32 MiB。
 - 当使用全局排序导入数据时，如果 TiDB 集群在导入任务尚未完成时被删除了，Amazon S3 上可能会残留用于全局排序的临时数据。该场景需要手动删除这些数据，以免增加 S3 存储成本。
 - 所需导入的数据不能存在主键或非空唯一索引冲突的记录，否则会导致任务失败。
+- 已经在运行的导入任务不支持被调度到新的 TiDB 节点上执行。当前在执行导入任务的 TiDB 节点如果重启，该 TiDB 节点不会再执行该导入任务，而是被转移到其他 TiDB 节点继续执行。
```
Comment (D3Hunter) [inline sql-statements/sql-statement-import-into.md:35]: server disk 导入的不会迁移

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

File: sql-statements/sql-statement-import-into.md (hunk 5 above)
Comment (Frank945946) [inline sql-statements/sql-statement-import-into.md:35]:
```suggestion
- 基于分布式框架调度的 ‘Import Into’ 任务，如该任务已运行，不支持被调度到新的 TiDB 节点上执行。当前在执行导入任务的 TiDB 节点如果重启，该 TiDB 节点不会再执行该导入任务，而是被转移到其他 TiDB 节点继续执行。
```

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-import-into.md [hunk 6]
```diff
@@ -32,6 +32,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - 当使用[全局排序](/tidb-global-sort.md)导入数据时，单行数据的总长度不能超过 32 MiB。
 - 当使用全局排序导入数据时，如果 TiDB 集群在导入任务尚未完成时被删除了，Amazon S3 上可能会残留用于全局排序的临时数据。该场景需要手动删除这些数据，以免增加 S3 存储成本。
 - 所需导入的数据不能存在主键或非空唯一索引冲突的记录，否则会导致任务失败。
+- 对于基于分布式执行框架调度的 `IMPORT INTO` 任务，如该任务已运行，不支持被调度到新的 TiDB 节点上执行。当前在执行导入任务的 TiDB 节点如果重启，该 TiDB 节点不会再执行该导入任务，而是被转移到其他 TiDB 节点继续执行。
```
Comment (Frank945946) [inline sql-statements/sql-statement-import-into.md:35]:
```suggestion
- 对于基于分布式执行框架调度的 `IMPORT INTO` 任务，如该任务已运行，不支持被调度到新的 TiDB 节点上执行。当前在执行导入任务的 TiDB 节点如果重启，该 TiDB 节点不会再执行该导入任务，而是被转移到其他 TiDB 节点继续执行。如果是导入 TiDB 节点本地的数据，任务异常后不会被 failover 到其他 TiDB 节点。
```

Comment (ran-huang) [inline sql-statements/sql-statement-import-into.md:35]:
```suggestion
- 对于基于分布式执行框架调度的 `IMPORT INTO` 任务，如该任务已运行，则不支持将其调度到新的 TiDB 节点上执行。当前在执行导入任务的 TiDB 节点如果重启，该 TiDB 节点不会再执行该导入任务，而是被转移到其他 TiDB 节点继续执行。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**